examples/python/translation_example.py

```
@@ -0,0 +1,227 @@
+    print("Search results from standard pipeline:")
+    for result in search_results[:2]:  # Show first 2 results
+        print(f"- {result}")
+    print()
+
+    # Reset for second demo
+    print("Resetting data for translation demo...")
+    await cognee.prune.prune_data()
+    await cognee.prune.prune_system(metadata=True)
+    
+    # Re-add the texts
+    for i, text in enumerate(multilingual_texts, 1):
```

Ruff (0.12.2)

Undefined name `multilingual_texts`

(F821)